.changeset/unlucky-ladybugs-laugh.md

```diff
@@ -0,0 +1,6 @@
+---
+"@paprika/data-grid": patch
+"@paprika/l10n": patch
```

Comment:
@allison-c Will this result in a bump to the `peerDependencies` of the `data-grid`? Or will that need to be updated manually? 

When the L10n team makes their subsequent update, are the dependencies bumped automatically as part of that process?

Reply:
Yes, bumping `@paprika/l10n` will bump all the packages having it as a peer dependency. However, we don't need to bump data-grid once again when l10n is bumped, cause they're releasing at the same time

When the L10n team makes their subsequent update, we should help them to add the `changeset` on their PR, so the @paprika/l10n can bump again and almost all the packages(since they have l10n as peerdependency) will bump again